scanner/src/main/kotlin/scanners/fossid/FossIdScanResults.kt

```diff
@@ -26,6 +26,33 @@ import org.ossreviewtoolkit.clients.fossid.model.summary.Summarizable
 import org.ossreviewtoolkit.model.CopyrightFinding
```

Comment:
Commit message: I would not mention parantheses at all, but just state that FossID can return a number of licenses whose identifiers cannot be parsed to SPDX expressions, and, therefore, a mapping is needed.